flaky test - CheckpointTest.testSuspendResumeWalPurgeJob

### To reproduce

CI, linux-arm64
```
2024-09-04T15:22:21.7053450Z 2024-09-04T15:22:21.697446Z I i.q.t.AbstractTest Starting test CheckpointTest#testSuspendResumeWalPurgeJob
2024-09-04T15:22:21.7054517Z 2024-09-04T15:22:21.697961Z I i.q.c.TableNameRegistryStore reloading tables file [path=/tmp/junit17631071340579173425/dbRoot/tables.d.0, threadId=16424]
2024-09-04T15:22:21.7055552Z random seeds: 1108247290386L, 2024-09-04T15:22:21.698049Z I i.q.t.AbstractCairoTest random seeds: 1108247290386L, 1725463341698L
2024-09-04T15:22:21.7056220Z 1725463341698L
2024-09-04T15:22:21.7084369Z 2024-09-04T15:22:21.707658Z I i.q.g.e.QueryProgress exe [id=0, sql=`create table testSuspendResumeWalPurgeJob as (select x,  timestamp_sequence('2022-02-24', 1000000L) ts  from long_sequence(5)) timestamp(ts) partition by DAY WAL`, principal=admin, cache=false, jit=false]
2024-09-04T15:22:21.7088132Z 2024-09-04T15:22:21.707926Z I i.q.c.CairoEngine locked [table=`testSuspendResumeWalPurgeJob~1`, thread=16424]
2024-09-04T15:22:21.7287826Z 2024-09-04T15:22:21.728219Z I i.q.c.p.WriterPool created [table=`testSuspendResumeWalPurgeJob~1`, thread=16424]
2024-09-04T15:22:21.7291188Z 2024-09-04T15:22:21.728241Z I i.q.c.TableWriter open 'testSuspendResumeWalPurgeJob'
2024-09-04T15:22:21.7319278Z 2024-09-04T15:22:21.731503Z I i.q.c.CairoEngine unlocked [table=`testSuspendResumeWalPurgeJob`]
2024-09-04T15:22:21.7335734Z 2024-09-04T15:22:21.733202Z I i.q.c.w.WalWriter open 'testSuspendResumeWalPurgeJob~1'
2024-09-04T15:22:21.7347578Z 2024-09-04T15:22:21.734230Z I i.q.c.w.WalWriter opened WAL segment [path=/testSuspendResumeWalPurgeJob~1/wal1/0/_event.i']
2024-09-04T15:22:21.7371609Z 2024-09-04T15:22:21.736094Z I i.q.c.w.WalWriter committed data block [wal=/testSuspendResumeWalPurgeJob~1/wal1/0, segmentTxn=0, seqTxn=1, rowLo=0, roHi=5, minTs=2022-02-24T00:00:00.000000Z, maxTs=2022-02-24T00:00:04.000000Z]
2024-09-04T15:22:21.7373165Z 2024-09-04T15:22:21.736146Z I i.q.g.e.QueryProgress fin [id=0, sql=`create table testSuspendResumeWalPurgeJob as (select x,  timestamp_sequence('2022-02-24', 1000000L) ts  from long_sequence(5)) timestamp(ts) partition by DAY WAL`, principal=admin, cache=false, jit=false, time=0]
2024-09-04T15:22:21.7445008Z 2024-09-04T15:22:21.737598Z I i.q.c.p.WriterPool >> [table=`testSuspendResumeWalPurgeJob~1`, thread=16424]
2024-09-04T15:22:21.7446730Z 2024-09-04T15:22:21.738112Z I i.q.c.TableWriter processing WAL [path=/testSuspendResumeWalPurgeJob~1/wal1/0, roLo=0, roHi=5, seqTxn=1, tsMin=2022-02-24T00:00:00.000000Z, tsMax=2022-02-24T00:00:04.000000Z, commitToTs=294247-01-10T04:00:54.775807Z]
2024-09-04T15:22:21.7447948Z 2024-09-04T15:22:21.738643Z I i.q.c.TableWriter switched partition [path=/testSuspendResumeWalPurgeJob~1/2022-02-24]
2024-09-04T15:22:21.7449131Z 2024-09-04T15:22:21.739145Z I i.q.c.w.ApplyWal2TableJob job finished [table=testSuspendResumeWalPurgeJob~1, seqTxn=1, transactions=1, rows=5, time=0ms, rate=5000000rows/s, physicalWrittenRowsMultiplier=1.0]
2024-09-04T15:22:21.7450202Z 2024-09-04T15:22:21.739175Z I i.q.c.p.WriterPool << [table=`testSuspendResumeWalPurgeJob~1`, thread=16424]
2024-09-04T15:22:21.7451200Z 2024-09-04T15:22:21.740351Z I i.q.g.e.QueryProgress exe [id=1, sql=`testSuspendResumeWalPurgeJob`, principal=admin, cache=false, jit=false]
2024-09-04T15:22:21.7452257Z 2024-09-04T15:22:21.740381Z I i.q.c.TableReader open partition [path=/testSuspendResumeWalPurgeJob~1/2022-02-24, rowCount=5, partitionIndex=0, partitionCount=1]
2024-09-04T15:22:21.7453312Z 2024-09-04T15:22:21.740969Z I i.q.g.e.QueryProgress fin [id=1, sql=`testSuspendResumeWalPurgeJob`, principal=admin, cache=false, jit=false, time=0]
2024-09-04T15:22:21.7454304Z 2024-09-04T15:22:21.743834Z I i.q.c.DatabaseCheckpointAgent creating table checkpoint [table=testSuspendResumeWalPurgeJob]
2024-09-04T15:22:21.7503990Z 2024-09-04T15:22:21.749875Z I i.q.c.DatabaseCheckpointAgent table included in the checkpoint [table=testSuspendResumeWalPurgeJob]
2024-09-04T15:22:21.8215749Z 2024-09-04T15:22:21.820856Z I i.q.c.DatabaseCheckpointAgent checkpoint created
2024-09-04T15:22:21.8217895Z 2024-09-04T15:22:21.820924Z I i.q.g.e.QueryProgress fin [id=-1, sql=`checkpoint create`, principal=admin, cache=false, jit=false, time=0]
2024-09-04T15:22:21.8242448Z 2024-09-04T15:22:21.823107Z I i.q.t.s.TestFilesFacadeImpl cannot remove, file is open: /tmp/junit17631071340579173425/dbRoot/testSuspendResumeWalPurgeJob~1.lock, fd=85900308268
2024-09-04T15:22:21.8244417Z 2024-09-04T15:22:21.823268Z I i.q.c.TableWriter closed 'testSuspendResumeWalPurgeJob'
2024-09-04T15:22:21.8245735Z 2024-09-04T15:22:21.823272Z I i.q.c.p.WriterPool closed [table=`testSuspendResumeWalPurgeJob~1`, reason=IDLE, by=16424]
2024-09-04T15:22:21.8247062Z 2024-09-04T15:22:21.823416Z I i.q.c.p.ReaderPool closed 'testSuspendResumeWalPurgeJob~1' [at=0:0, reason=IDLE]
2024-09-04T15:22:21.8254735Z 2024-09-04T15:22:21.823573Z I i.q.c.w.s.TableSequencerAPI releasing idle table sequencer [tableDir=testSuspendResumeWalPurgeJob~1]
2024-09-04T15:22:21.8256319Z 2024-09-04T15:22:21.823794Z I i.q.c.w.WalWriter closed 'testSuspendResumeWalPurgeJob'
2024-09-04T15:22:21.8257600Z 2024-09-04T15:22:21.823798Z I i.q.c.p.WalWriterPool closed 'testSuspendResumeWalPurgeJob~1' [at=0:0, reason=IDLE]
2024-09-04T15:22:21.8264307Z 2024-09-04T15:22:21.825989Z I i.q.g.e.QueryProgress fin [id=-1, sql=`checkpoint release`, principal=admin, cache=false, jit=false, time=0]
2024-09-04T15:22:21.8309664Z 2024-09-04T15:22:21.827203Z E i.q.t.AbstractCairoTest Error in test: 
2024-09-04T15:22:21.8311365Z java.lang.AssertionError: /tmp/junit17631071340579173425/dbRoot/testSuspendResumeWalPurgeJob~1/wal1/0 expected:<false> but was:<true>
2024-09-04T15:22:21.8312412Z 	at org.junit.Assert.fail(Assert.java:89)
2024-09-04T15:22:21.8313656Z 	at org.junit.Assert.failNotEquals(Assert.java:835)
2024-09-04T15:22:21.8314679Z 	at org.junit.Assert.assertEquals(Assert.java:120)
2024-09-04T15:22:21.8315599Z 	at io.questdb.test.AbstractCairoTest.assertSegmentExistence(AbstractCairoTest.java:1750)
2024-09-04T15:22:21.8316550Z 	at io.questdb.test.AbstractCairoTest.assertSegmentExistence(AbstractCairoTest.java:1743)
2024-09-04T15:22:21.8317551Z 	at io.questdb.test.griffin.CheckpointTest.lambda$testSuspendResumeWalPurgeJob$40(CheckpointTest.java:1074)
2024-09-04T15:22:21.8318543Z 	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$8(AbstractCairoTest.java:977)
2024-09-04T15:22:21.8319461Z 	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:584)
2024-09-04T15:22:21.8320350Z 	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:974)
2024-09-04T15:22:21.8321258Z 	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:957)
2024-09-04T15:22:21.8322234Z 	at io.questdb.test.griffin.CheckpointTest.testSuspendResumeWalPurgeJob(CheckpointTest.java:1014)
2024-09-04T15:22:21.8323154Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2024-09-04T15:22:21.8324065Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2024-09-04T15:22:21.8325031Z 	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2024-09-04T15:22:21.8325957Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2024-09-04T15:22:21.8326851Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2024-09-04T15:22:21.8327809Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2024-09-04T15:22:21.8328745Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2024-09-04T15:22:21.8329685Z 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2024-09-04T15:22:21.8330607Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2024-09-04T15:22:21.8331528Z 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2024-09-04T15:22:21.8332754Z 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
2024-09-04T15:22:21.8333664Z 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
2024-09-04T15:22:21.8334654Z 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
2024-09-04T15:22:21.8340991Z 	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
2024-09-04T15:22:21.8342073Z 	at java.lang.Thread.run(Thread.java:829)
2024-09-04T15:22:21.8359124Z 2024-09-04T15:22:21.834796Z I i.q.t.AbstractCairoTest Tearing down test CheckpointTest#testSuspendResumeWalPurgeJob
```

### QuestDB version:

90e1bc528d0ddea66fa48db40d36276f834ef1a6

### OS, in case of Docker specify Docker and the Host OS:

Linux ARM64

### File System, in case of Docker specify Host File System:

not sure

### Full Name:

Jaromir Hamala

### Affiliation:

QuestDB

### Have you followed Linux, MacOs kernel configuration steps to increase Maximum open files and Maximum virtual memory areas limit?

- [X] Yes, I have

### Additional context

_No response_

--- a/core/src/test/java/io/questdb/test/griffin/CheckpointTest.java
+++ b/core/src/test/java/io/questdb/test/griffin/CheckpointTest.java
@@ -44,7 +44,6 @@ import io.questdb.std.str.Path;
 import io.questdb.std.str.Utf8s;
 import io.questdb.test.AbstractCairoTest;
 import io.questdb.test.std.TestFilesFacadeImpl;
-import io.questdb.test.tools.StationaryMicrosClock;
 import io.questdb.test.tools.TestUtils;
 import org.junit.*;
 
@@ -60,7 +59,6 @@ public class CheckpointTest extends AbstractCairoTest {
 
     @BeforeClass
     public static void setUpStatic() throws Exception {
-        testMicrosClock = StationaryMicrosClock.INSTANCE;
         path = new Path();
         triggerFilePath = new Path();
         ff = testFilesFacade;
@@ -613,6 +611,7 @@ public class CheckpointTest extends AbstractCairoTest {
     @Test
     public void testCheckpointStatus() throws Exception {
         assertMemoryLeak(() -> {
+            currentMicros = 0;
             assertSql(
                     "in_progress\tstarted_at\n" +
                             "false\t\n",

